lib/logstash/outputs/elasticsearch.rb

```diff
@@ -62,6 +62,8 @@
 #   the max interval time is reached, which is set in :retry_max_interval.
 # - Events which are not retryable or have reached their max retry count are logged to stderr.
 class LogStash::Outputs::ElasticSearch < LogStash::Outputs::Base
+  attr_reader :client
```

Comment:
That was added in for testing. I could use an attr getter, but I think it's generally better to add an accessor.